manuscript.Rmd

```diff
@@ -980,6 +981,7 @@ The code for this power analysis can be found in the [online supplementary mater
 
 Monte Carlo simulations are, of course, not only applicable for this analysis method and also allow researchers to investigate other relevant properties of their analysis method beyond power [@brandmaier2018precision; @harrisonIntroductionMonteCarlo2010; @raychaudhuriIntroductionMonteCarlo2008].
 
+<!--AB this code will not be understood by most of the readers? Can we remove this? Instead keep the results text below but explain what the variables choosen_power and minn mean and that they were either set in the simulation or simulation results #necessary-->
```

Comment:
Hey @brandmaier this code is hidden (echo=FALSE).

Reply:
Hm... I rendered the document outside Docker and this showed up in the PDF. Not sure why. Anyhow, then everything is fine.

Reply:
Curious and very weird that knitr would malfunction like that, maybe there was a typo or something in this particular chunks options.

Reply:
<img width="899" alt="Screenshot 2021-08-05 at 21 18 55" src="https://user-images.githubusercontent.com/116031/128408588-b8bf2b26-194f-4120-a30b-49d49a291ef2.png">

This is what it looks like.

Reply:
True! This is a very similar code I forgot that it was still in there.